membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -1,40 +1,37 @@
 package controllers
```

Comment:
This controller is getting pretty large - we could consider moving the functions related to payment method update  into their own controller?

Reply:
this might be a refactor for future PRs but I think we could break down this into more specific categories of functionality. 
Especially the big functions like cancelSubscription or updateDirectDebit which contain nested functions inside

Reply:
yeah discussed the same with Jacob and going to do that in a PR immediately following this one, to stop this one from being too large